elide-core/src/main/java/com/yahoo/elide/core/security/checks/OperationCheck.java

```diff
@@ -31,4 +31,13 @@
      * @return true if security check passed
      */
     public abstract boolean ok(T object, RequestScope requestScope, Optional<ChangeSpec> changeSpec);
+
+    /**
+     * Should check run inline or be deferred to Commit
+     *
+     * @return true to run at commit
+     */
+    public boolean runAtCommit() {
```

Comment:
Can we move this into Check as default method?

Then, in FilterExpressionCheck and UserCheck, override it to always return false and make the method final (so it cannot be overwritten).